azure-py-call-azure-sdk/__main__.py

```diff
@@ -0,0 +1,52 @@
+"""A program to demonstrate accessing Azure Python SDK"""
+
+from azure.core.credentials import AccessToken
+from azure.mgmt.authorization import AuthorizationManagementClient
+from pulumi_azure_native import authorization, containerregistry, resources
+
+
+class TokenCred:
+    def __init__(self, token):
+        self.token = token
+
+    def get_token(self, *scopes, **kwargs) -> 'AccessToken':
```

Comment:
This is the simplest mechanism I could find to inject the token. Very few examples and documentation on this from what I could see.